packages/io/json-serializer/README.md

```diff
@@ -1,15 +1,19 @@
 ## @jscad/json-serializer
 
-> json serializer for the jscad project (from CSG)
+> Serializer of JSCAD geometries to JSON notation
```

Comment:
JSON notation is redundant simply JSON is fine :)

Reply:
Done